ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ranger/RangerRestClientImpl.java

```diff
@@ -79,6 +74,7 @@
   private static final String RANGER_REST_URL_EXPORTJSONFILE = "service/plugins/policies/exportJson";
   private static final String RANGER_REST_URL_IMPORTJSONFILE =
       "service/plugins/policies/importPoliciesFromFile";
+  private static final String RANGER_REST_URL_DELETEPOLICY = "service/public/v2/api/policy";
```

Comment:
is this not available as part of service/plugins/policies?

Reply:
No, This method is not available.

Reply:
Please cross check with Ranger team/public docs

Reply:
ok